plugins/language_fsharp.lua

```diff
@@ -0,0 +1,108 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "F#",
+  files = { "%.fsi?$" },
+  comment = "//",
```

Comment:
Seems like it supports block comments with `(*` and `*)`.

Reply:
Thanks, I'll add it right away.